### PR TITLE
Use Mono.Android as a reference for netstandard facade

### PR DESCRIPTION
**Pull request by EgorBo**
see https://github.com/mono/mono/pull/10835

### CODE REVIEW

File: mcs/class/Facades/netstandard/Makefile [hunk 1]
```diff
@@ -41,13 +41,17 @@ EMBEDDED_DRAWING_DEP := $(filter xammac_net_4_5 testing_aot_full testing_aot_hyb
 
 ifdef EMBEDDED_DRAWING_DEP
 LIB_REFS += Facades/System.Drawing.Primitives
+else 
+ifeq ($(PROFILE),monodroid)
+LIB_REFS += Mono.Android
```
Comment (marek-safar) [inline mcs/class/Facades/netstandard/Makefile]: This should be `API_BIN_REFS = Mono.Android`

Comment (marek-safar) [reply mcs/class/Facades/netstandard/Makefile]: also move it out of `EMBEDDED_DRAWING_DEP`

File: mcs/class/Facades/subdirs.make [hunk 2]
```diff
@@ -41,6 +41,9 @@ netstandard_drawing_SUBDIRS = System.Drawing.Primitives netstandard
 monotouch_SUBDIRS = $(common_DEPS_SUBDIRS) $(mobile_only_DEPS_SUBDIRS)
 monotouch_PARALLEL_SUBDIRS = $(common_SUBDIRS) $(mobile_only_SUBDIRS)
 
+monodroid_SUBDIRS = $(monotouch_SUBDIRS) netstandard
```
Comment (akoeplinger) [inline mcs/class/Facades/subdirs.make]: we should do the Mono.Android reference changes for System.Drawing.Primitives too and then just use `monodroid_SUBDIRS = $(monotouch_SUBDIRS) $(netstandard_drawing_SUBDIRS)` here

Comment (akoeplinger) [reply mcs/class/Facades/subdirs.make]: Sure, that works too. Though netstandard might require more types in the future (I've seen plans somewhere) that won't ever be in System.Drawing.Primitives so I'd prefer just refing Mono.Android.dll from netstandard.dll.

Comment (marek-safar) [reply mcs/class/Facades/subdirs.make]: Let's just use it `System.Drawing.Primitives` that one we can easily fix also XI and we are more compatible with official package

**Comment by EgorBo** [comment]
@akoeplinger @marek-safar Any ideas why `netstandard` fails to build on `monodroid` profile:
```
TypeForwarders.cs(875,81): error CS0234: The type or namespace name 'Color' does not exist in the namespace 'System.Drawing' (are you missing an assembly reference?)
TypeForwarders.cs(876,81): error CS0234: The type or namespace name 'Point' does not exist in the namespace 'System.Drawing' (are you missing an assembly reference?)
TypeForwarders.cs(877,81): error CS0234: The type or namespace name 'PointF' does not exist in the namespace 'System.Drawing' (are you missing an assembly reference?)
TypeForwarders.cs(878,81): error CS0234: The type or namespace name 'Rectangle' does not exist in the namespace 'System.Drawing' (are you missing an assembly reference?)
TypeForwarders.cs(879,81): error CS0234: The type or namespace name 'RectangleF' does not exist in the namespace 'System.Drawing' (are you missing an assembly reference?)
TypeForwarders.cs(880,81): error CS0234: The type or namespace name 'Size' does not exist in the namespace 'System.Drawing' (are you missing an assembly reference?)
TypeForwarders.cs(881,81): error CS0234: The type or namespace name 'SizeF' does not exist in the namespace 'System.Drawing' (are you missing an assembly reference?)
```
so this line `LIB_REFS += Facades/System.Drawing.Primitives` doesn't bring those types

**Comment by marek-safar** [comment]
@monojenkins backport 2018-10

**Comment by marek-safar** [comment]
@monojenkins backport 2018-08

**Comment by marek-safar** [comment]
@monojenkins commit csproj and apidiff

**Pull request merged**